JuliaGPU/session2/Gauri/q2.jl

```diff
@@ -0,0 +1,24 @@
+using Test
+#=   =#
+function bs_sqrt(x)
```

Comment:
You haven't implemented the algorithm correctly. Please look into the link in assignment 2 q2 for more details.